[BUG] Error compiling MMU2

### Description

Compiling with MMU2 isnt possible anymore. 

Download the newest marlin and activate mmu2 support and activate #define MMU2_MENUS  . Succesfully Compiling isnt possible anymore.
Following error message will come up
`In file included from Marlin\src\lcd\menu\menu_mmu2.cpp:29:0:
Marlin\src\lcd\menu\menu_mmu2.cpp: In function 'void menu_mmu2_load_filament()':
Marlin\src\lcd\menu\menu_mmu2.cpp:66:15: error: 'MSG_MMU2_FILAMENT0' is not a member of 'Language_en'
   ACTION_ITEM(MSG_MMU2_FILAMENT0, []{ _mmu2_load_filament(0); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:66:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT0, []{ _mmu2_load_filament(0); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:66:15: note: suggested alternative: 'MSG_MMU2_FILAMENT_N'
   ACTION_ITEM(MSG_MMU2_FILAMENT0, []{ _mmu2_load_filament(0); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:66:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT0, []{ _mmu2_load_filament(0); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:67:15: error: 'MSG_MMU2_FILAMENT1' is not a member of 'Language_en'
   ACTION_ITEM(MSG_MMU2_FILAMENT1, []{ _mmu2_load_filament(1); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:67:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT1, []{ _mmu2_load_filament(1); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:67:15: note: suggested alternative: 'MSG_MMU2_FILAMENT_N'
   ACTION_ITEM(MSG_MMU2_FILAMENT1, []{ _mmu2_load_filament(1); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:67:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT1, []{ _mmu2_load_filament(1); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:68:15: error: 'MSG_MMU2_FILAMENT2' is not a member of 'Language_en'
   ACTION_ITEM(MSG_MMU2_FILAMENT2, []{ _mmu2_load_filament(2); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:68:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT2, []{ _mmu2_load_filament(2); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:68:15: note: suggested alternative: 'MSG_MMU2_FILAMENT_N'
   ACTION_ITEM(MSG_MMU2_FILAMENT2, []{ _mmu2_load_filament(2); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:68:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT2, []{ _mmu2_load_filament(2); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:69:15: error: 'MSG_MMU2_FILAMENT3' is not a member of 'Language_en'
   ACTION_ITEM(MSG_MMU2_FILAMENT3, []{ _mmu2_load_filament(3); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:69:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT3, []{ _mmu2_load_filament(3); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:69:15: note: suggested alternative: 'MSG_MMU2_FILAMENT_N'
   ACTION_ITEM(MSG_MMU2_FILAMENT3, []{ _mmu2_load_filament(3); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:69:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT3, []{ _mmu2_load_filament(3); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:70:15: error: 'MSG_MMU2_FILAMENT4' is not a member of 'Language_en'
   ACTION_ITEM(MSG_MMU2_FILAMENT4, []{ _mmu2_load_filament(4); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:70:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT4, []{ _mmu2_load_filament(4); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:70:15: note: suggested alternative: 'MSG_MMU2_FILAMENT_N'
   ACTION_ITEM(MSG_MMU2_FILAMENT4, []{ _mmu2_load_filament(4); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:70:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT4, []{ _mmu2_load_filament(4); });
   ^~~~~~~~~~~
compilation terminated due to -fmax-errors=5.
*** [.pio\build\BIGTREE_SKR_PRO\src\src\lcd\menu\menu_mmu2.cpp.o] Error 1`

if you disable #define MMU2_MENUS 
following error message will come up 

`Marlin\src\gcode\feature\pause\M701_M702.cpp: In static member function 'static void GcodeSuite::M701()':
Marlin\src\gcode\feature\pause\M701_M702.cpp:104:10: error: 'class MMU2' has no member named 'load_filament_to_nozzle'; did you mean 'set_filament_type'?
     mmu2.load_filament_to_nozzle(target_extruder);
          ^~~~~~~~~~~~~~~~~~~~~~~
          set_filament_type
Marlin\src\gcode\feature\pause\M701_M702.cpp: In static member function 'static void GcodeSuite::M702()':
Marlin\src\gcode\feature\pause\M701_M702.cpp:208:10: error: 'class MMU2' has no member named 'unload'
     mmu2.unload();
          ^~~~~~
*** [.pio\build\BIGTREE_SKR_PRO\src\src\gcode\feature\pause\M701_M702.cpp.o] Error 1
[ConfigurationH.zip](https://github.com/MarlinFirmware/Marlin/files/3855837/ConfigurationH.zip)
`

made the changes  
this error comes with enabled  #define MMU2_MENUS
`In file included from Marlin\src\lcd\menu\menu_mmu2.cpp:29:0:
Marlin\src\lcd\menu\menu_mmu2.cpp: In lambda function:
Marlin\src\lcd\menu\menu_mmu2.cpp:77:45: error: '_mmu2_eject_filament' was not declared in this scope
   ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(0); });
                                             ^
Marlin\src\lcd\menu\menu.h:370:39: note: in definition of macro '_MENU_INNER_P'
     MenuItem_##TYPE::action(plabel, ##V);                     \
                                       ^
Marlin\src\lcd\menu\menu.h:427:52: note: in expansion of macro '_MENU_ITEM_N_P'
 #define MENU_ITEM_N_P(TYPE, N, PLABEL, V...)       _MENU_ITEM_N_P(TYPE, N, false, PLABEL, ##V)
                                                    ^~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:435:53: note: in expansion of macro 'MENU_ITEM_N_P'
 #define ACTION_ITEM_N_P(N, PLABEL, ACTION)          MENU_ITEM_N_P(function, N, PLABEL, ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:436:51: note: in expansion of macro 'ACTION_ITEM_N_P'
 #define ACTION_ITEM_N(N, LABEL, ACTION)           ACTION_ITEM_N_P(N, GET_TEXT(LABEL), ACTION)
                                                   ^~~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:77:3: note: in expansion of macro 'ACTION_ITEM_N'
   ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(0); });
   ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:77:45: note: suggested alternative: '_mmu2_load_filament'
   ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(0); });
                                             ^
Marlin\src\lcd\menu\menu.h:370:39: note: in definition of macro '_MENU_INNER_P'
     MenuItem_##TYPE::action(plabel, ##V);                     \
                                       ^
Marlin\src\lcd\menu\menu.h:427:52: note: in expansion of macro '_MENU_ITEM_N_P'
 #define MENU_ITEM_N_P(TYPE, N, PLABEL, V...)       _MENU_ITEM_N_P(TYPE, N, false, PLABEL, ##V)
                                                    ^~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:435:53: note: in expansion of macro 'MENU_ITEM_N_P'
 #define ACTION_ITEM_N_P(N, PLABEL, ACTION)          MENU_ITEM_N_P(function, N, PLABEL, ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:436:51: note: in expansion of macro 'ACTION_ITEM_N_P'
 #define ACTION_ITEM_N(N, LABEL, ACTION)           ACTION_ITEM_N_P(N, GET_TEXT(LABEL), ACTION)
                                                   ^~~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:77:3: note: in expansion of macro 'ACTION_ITEM_N'
   ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(0); });
   ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp: In lambda function:
Marlin\src\lcd\menu\menu_mmu2.cpp:77:45: error: '_mmu2_eject_filament' was not declared in this scope
   ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(0); });
                                             ^
Marlin\src\lcd\menu\menu.h:375:58: note: in definition of macro '_MENU_INNER_P'
       (encoderLine == _thisItemNr, _lcdLineNr, plabel, ##V);  \
                                                          ^
Marlin\src\lcd\menu\menu.h:427:52: note: in expansion of macro '_MENU_ITEM_N_P'
 #define MENU_ITEM_N_P(TYPE, N, PLABEL, V...)       _MENU_ITEM_N_P(TYPE, N, false, PLABEL, ##V)
                                                    ^~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:435:53: note: in expansion of macro 'MENU_ITEM_N_P'
 #define ACTION_ITEM_N_P(N, PLABEL, ACTION)          MENU_ITEM_N_P(function, N, PLABEL, ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:436:51: note: in expansion of macro 'ACTION_ITEM_N_P'
 #define ACTION_ITEM_N(N, LABEL, ACTION)           ACTION_ITEM_N_P(N, GET_TEXT(LABEL), ACTION)
                                                   ^~~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:77:3: note: in expansion of macro 'ACTION_ITEM_N'
   ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(0); });
   ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:77:45: note: suggested alternative: '_mmu2_load_filament'
   ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(0); });
                                             ^
Marlin\src\lcd\menu\menu.h:375:58: note: in definition of macro '_MENU_INNER_P'
       (encoderLine == _thisItemNr, _lcdLineNr, plabel, ##V);  \
                                                          ^
Marlin\src\lcd\menu\menu.h:427:52: note: in expansion of macro '_MENU_ITEM_N_P'
 #define MENU_ITEM_N_P(TYPE, N, PLABEL, V...)       _MENU_ITEM_N_P(TYPE, N, false, PLABEL, ##V)
                                                    ^~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:435:53: note: in expansion of macro 'MENU_ITEM_N_P'
 #define ACTION_ITEM_N_P(N, PLABEL, ACTION)          MENU_ITEM_N_P(function, N, PLABEL, ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:436:51: note: in expansion of macro 'ACTION_ITEM_N_P'
 #define ACTION_ITEM_N(N, LABEL, ACTION)           ACTION_ITEM_N_P(N, GET_TEXT(LABEL), ACTION)
                                                   ^~~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:77:3: note: in expansion of macro 'ACTION_ITEM_N'
   ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(0); });
   ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp: In lambda function:
Marlin\src\lcd\menu\menu_mmu2.cpp:78:45: error: '_mmu2_eject_filament' was not declared in this scope
   ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(1); });
                                             ^
Marlin\src\lcd\menu\menu.h:370:39: note: in definition of macro '_MENU_INNER_P'
     MenuItem_##TYPE::action(plabel, ##V);                     \
                                       ^
Marlin\src\lcd\menu\menu.h:427:52: note: in expansion of macro '_MENU_ITEM_N_P'
 #define MENU_ITEM_N_P(TYPE, N, PLABEL, V...)       _MENU_ITEM_N_P(TYPE, N, false, PLABEL, ##V)
                                                    ^~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:435:53: note: in expansion of macro 'MENU_ITEM_N_P'
 #define ACTION_ITEM_N_P(N, PLABEL, ACTION)          MENU_ITEM_N_P(function, N, PLABEL, ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:436:51: note: in expansion of macro 'ACTION_ITEM_N_P'
 #define ACTION_ITEM_N(N, LABEL, ACTION)           ACTION_ITEM_N_P(N, GET_TEXT(LABEL), ACTION)
                                                   ^~~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:78:3: note: in expansion of macro 'ACTION_ITEM_N'
   ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(1); });
   ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:78:45: note: suggested alternative: '_mmu2_load_filament'
   ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(1); });
                                             ^
Marlin\src\lcd\menu\menu.h:370:39: note: in definition of macro '_MENU_INNER_P'
     MenuItem_##TYPE::action(plabel, ##V);                     \
                                       ^
Marlin\src\lcd\menu\menu.h:427:52: note: in expansion of macro '_MENU_ITEM_N_P'
 #define MENU_ITEM_N_P(TYPE, N, PLABEL, V...)       _MENU_ITEM_N_P(TYPE, N, false, PLABEL, ##V)
                                                    ^~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:435:53: note: in expansion of macro 'MENU_ITEM_N_P'
 #define ACTION_ITEM_N_P(N, PLABEL, ACTION)          MENU_ITEM_N_P(function, N, PLABEL, ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:436:51: note: in expansion of macro 'ACTION_ITEM_N_P'
 #define ACTION_ITEM_N(N, LABEL, ACTION)           ACTION_ITEM_N_P(N, GET_TEXT(LABEL), ACTION)
                                                   ^~~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:78:3: note: in expansion of macro 'ACTION_ITEM_N'
   ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(1); });
   ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp: In lambda function:
Marlin\src\lcd\menu\menu_mmu2.cpp:78:45: error: '_mmu2_eject_filament' was not declared in this scope
   ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(1); });
                                             ^
Marlin\src\lcd\menu\menu.h:375:58: note: in definition of macro '_MENU_INNER_P'
       (encoderLine == _thisItemNr, _lcdLineNr, plabel, ##V);  \
                                                          ^
Marlin\src\lcd\menu\menu.h:427:52: note: in expansion of macro '_MENU_ITEM_N_P'
 #define MENU_ITEM_N_P(TYPE, N, PLABEL, V...)       _MENU_ITEM_N_P(TYPE, N, false, PLABEL, ##V)
                                                    ^~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:435:53: note: in expansion of macro 'MENU_ITEM_N_P'
 #define ACTION_ITEM_N_P(N, PLABEL, ACTION)          MENU_ITEM_N_P(function, N, PLABEL, ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:436:51: note: in expansion of macro 'ACTION_ITEM_N_P'
 #define ACTION_ITEM_N(N, LABEL, ACTION)           ACTION_ITEM_N_P(N, GET_TEXT(LABEL), ACTION)
                                                   ^~~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:78:3: note: in expansion of macro 'ACTION_ITEM_N'
   ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(1); });
   ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:78:45: note: suggested alternative: '_mmu2_load_filament'
   ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(1); });
                                             ^
Marlin\src\lcd\menu\menu.h:375:58: note: in definition of macro '_MENU_INNER_P'
       (encoderLine == _thisItemNr, _lcdLineNr, plabel, ##V);  \
                                                          ^
Marlin\src\lcd\menu\menu.h:427:52: note: in expansion of macro '_MENU_ITEM_N_P'
 #define MENU_ITEM_N_P(TYPE, N, PLABEL, V...)       _MENU_ITEM_N_P(TYPE, N, false, PLABEL, ##V)
                                                    ^~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:435:53: note: in expansion of macro 'MENU_ITEM_N_P'
 #define ACTION_ITEM_N_P(N, PLABEL, ACTION)          MENU_ITEM_N_P(function, N, PLABEL, ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:436:51: note: in expansion of macro 'ACTION_ITEM_N_P'
 #define ACTION_ITEM_N(N, LABEL, ACTION)           ACTION_ITEM_N_P(N, GET_TEXT(LABEL), ACTION)
                                                   ^~~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:78:3: note: in expansion of macro 'ACTION_ITEM_N'
   ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(1); });
   ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp: In lambda function:
Marlin\src\lcd\menu\menu_mmu2.cpp:79:45: error: '_mmu2_eject_filament' was not declared in this scope
   ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(2); });
                                             ^
Marlin\src\lcd\menu\menu.h:370:39: note: in definition of macro '_MENU_INNER_P'
     MenuItem_##TYPE::action(plabel, ##V);                     \
                                       ^
Marlin\src\lcd\menu\menu.h:427:52: note: in expansion of macro '_MENU_ITEM_N_P'
 #define MENU_ITEM_N_P(TYPE, N, PLABEL, V...)       _MENU_ITEM_N_P(TYPE, N, false, PLABEL, ##V)
                                                    ^~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:435:53: note: in expansion of macro 'MENU_ITEM_N_P'
 #define ACTION_ITEM_N_P(N, PLABEL, ACTION)          MENU_ITEM_N_P(function, N, PLABEL, ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:436:51: note: in expansion of macro 'ACTION_ITEM_N_P'
 #define ACTION_ITEM_N(N, LABEL, ACTION)           ACTION_ITEM_N_P(N, GET_TEXT(LABEL), ACTION)
                                                   ^~~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:79:3: note: in expansion of macro 'ACTION_ITEM_N'
   ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(2); });
   ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:79:45: note: suggested alternative: '_mmu2_load_filament'
   ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(2); });
                                             ^
Marlin\src\lcd\menu\menu.h:370:39: note: in definition of macro '_MENU_INNER_P'
     MenuItem_##TYPE::action(plabel, ##V);                     \
                                       ^
Marlin\src\lcd\menu\menu.h:427:52: note: in expansion of macro '_MENU_ITEM_N_P'
 #define MENU_ITEM_N_P(TYPE, N, PLABEL, V...)       _MENU_ITEM_N_P(TYPE, N, false, PLABEL, ##V)
                                                    ^~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:435:53: note: in expansion of macro 'MENU_ITEM_N_P'
 #define ACTION_ITEM_N_P(N, PLABEL, ACTION)          MENU_ITEM_N_P(function, N, PLABEL, ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:436:51: note: in expansion of macro 'ACTION_ITEM_N_P'
 #define ACTION_ITEM_N(N, LABEL, ACTION)           ACTION_ITEM_N_P(N, GET_TEXT(LABEL), ACTION)
                                                   ^~~~~~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:79:3: note: in expansion of macro 'ACTION_ITEM_N'
   ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(2); });
   ^~~~~~~~~~~~~
compilation terminated due to -fmax-errors=5.
Compiling .pio\build\BIGTREE_SKR_PRO\src\src\libs\L6470\L6470_Marlin.cpp.o
*** [.pio\build\BIGTREE_SKR_PRO\src\src\lcd\menu\menu_mmu2.cpp.o] Error 1`

this error only enabled mmu2 support 
`Marlin\src\gcode\feature\pause\M701_M702.cpp: In static member function 'static void GcodeSuite::M701()':
Marlin\src\gcode\feature\pause\M701_M702.cpp:104:10: error: 'class MMU2' has no member named 'load_filament_to_nozzle'; did you mean 'set_filament_type'?
     mmu2.load_filament_to_nozzle(target_extruder);
          ^~~~~~~~~~~~~~~~~~~~~~~
          set_filament_type
Marlin\src\gcode\feature\pause\M701_M702.cpp: In static member function 'static void GcodeSuite::M702()':
Marlin\src\gcode\feature\pause\M701_M702.cpp:208:10: error: 'class MMU2' has no member named 'unload'
     mmu2.unload();
          ^~~~~~
*** [.pio\build\BIGTREE_SKR_PRO\src\src\gcode\feature\pause\M701_M702.cpp.o] Error 1`

The problem was my config. I had enabled FILAMENT_LOAD_UNLOAD_GCODE  in the conf.adv without this it compiles succesfully also without any changes. 
Maybe this function is not compatibel with the mmu2 feature
Edit: with enabled #define MMU2_MENUS comes this error again 

`In file included from Marlin\src\lcd\menu\menu_mmu2.cpp:29:0:
Marlin\src\lcd\menu\menu_mmu2.cpp: In function 'void menu_mmu2_load_filament()':
Marlin\src\lcd\menu\menu_mmu2.cpp:66:15: error: 'MSG_MMU2_FILAMENT0' is not a member of 'Language_en'
   ACTION_ITEM(MSG_MMU2_FILAMENT0, []{ _mmu2_load_filament(0); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:66:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT0, []{ _mmu2_load_filament(0); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:66:15: note: suggested alternative: 'MSG_MMU2_FILAMENT_N'
   ACTION_ITEM(MSG_MMU2_FILAMENT0, []{ _mmu2_load_filament(0); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:66:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT0, []{ _mmu2_load_filament(0); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:67:15: error: 'MSG_MMU2_FILAMENT1' is not a member of 'Language_en'
   ACTION_ITEM(MSG_MMU2_FILAMENT1, []{ _mmu2_load_filament(1); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:67:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT1, []{ _mmu2_load_filament(1); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:67:15: note: suggested alternative: 'MSG_MMU2_FILAMENT_N'
   ACTION_ITEM(MSG_MMU2_FILAMENT1, []{ _mmu2_load_filament(1); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:67:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT1, []{ _mmu2_load_filament(1); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:68:15: error: 'MSG_MMU2_FILAMENT2' is not a member of 'Language_en'
   ACTION_ITEM(MSG_MMU2_FILAMENT2, []{ _mmu2_load_filament(2); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:68:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT2, []{ _mmu2_load_filament(2); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:68:15: note: suggested alternative: 'MSG_MMU2_FILAMENT_N'
   ACTION_ITEM(MSG_MMU2_FILAMENT2, []{ _mmu2_load_filament(2); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
Compiling .pio\build\BIGTREE_SKR_PRO\src\src\libs\hex_print_routines.cpp.o
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:68:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT2, []{ _mmu2_load_filament(2); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:69:15: error: 'MSG_MMU2_FILAMENT3' is not a member of 'Language_en'
   ACTION_ITEM(MSG_MMU2_FILAMENT3, []{ _mmu2_load_filament(3); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:69:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT3, []{ _mmu2_load_filament(3); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:69:15: note: suggested alternative: 'MSG_MMU2_FILAMENT_N'
   ACTION_ITEM(MSG_MMU2_FILAMENT3, []{ _mmu2_load_filament(3); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
Compiling .pio\build\BIGTREE_SKR_PRO\src\src\libs\least_squares_fit.cpp.o
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:69:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT3, []{ _mmu2_load_filament(3); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:70:15: error: 'MSG_MMU2_FILAMENT4' is not a member of 'Language_en'
   ACTION_ITEM(MSG_MMU2_FILAMENT4, []{ _mmu2_load_filament(4); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:70:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT4, []{ _mmu2_load_filament(4); });
   ^~~~~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:70:15: note: suggested alternative: 'MSG_MMU2_FILAMENT_N'
   ACTION_ITEM(MSG_MMU2_FILAMENT4, []{ _mmu2_load_filament(4); });
               ^
Marlin\src\lcd\menu\menu.h:367:24: note: in definition of macro '_MENU_INNER_P'
   PGM_P const plabel = PLABEL;                                \
                        ^~~~~~
Marlin\src\lcd\menu\menu.h:424:54: note: in expansion of macro '_MENU_ITEM_P'
 #define MENU_ITEM_P(TYPE, PLABEL, V...)              _MENU_ITEM_P(TYPE, false, PLABEL, ##V)
                                                      ^~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:432:55: note: in expansion of macro 'MENU_ITEM_P'
 #define ACTION_ITEM_P(PLABEL, ACTION)                 MENU_ITEM_P(function, PLABEL, ACTION)
                                                       ^~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:53: note: in expansion of macro 'ACTION_ITEM_P'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                     ^~~~~~~~~~~~~
Marlin\src\lcd\menu\menu.h:433:67: note: in expansion of macro 'GET_TEXT'
 #define ACTION_ITEM(LABEL, ACTION)                  ACTION_ITEM_P(GET_TEXT(LABEL),  ACTION)
                                                                   ^~~~~~~~
Marlin\src\lcd\menu\menu_mmu2.cpp:70:3: note: in expansion of macro 'ACTION_ITEM'
   ACTION_ITEM(MSG_MMU2_FILAMENT4, []{ _mmu2_load_filament(4); });
   ^~~~~~~~~~~
compilation terminated due to -fmax-errors=5.
*** [.pio\build\BIGTREE_SKR_PRO\src\src\lcd\menu\menu_mmu2.cpp.o] Error 1`
i will try with your changes again

--- a/Marlin/src/lcd/menu/menu_mmu2.cpp
+++ b/Marlin/src/lcd/menu/menu_mmu2.cpp
@@ -39,7 +39,7 @@ void _mmu2_load_filamentToNozzle(uint8_t index) {
   ui.reset_status();
   ui.return_to_status();
   ui.status_printf_P(0,  GET_TEXT(MSG_MMU2_LOADING_FILAMENT), int(index + 1));
-  if (mmu2.load_filament_to_nozzle(index)) ui.reset_status();
+  if (mmu2.load_filament_toùozzle(index)) ui.reset_status();
 }
 
 inline void action_mmu2_load_filament_to_nozzle(const uint8_t tool) {
@@ -63,22 +63,22 @@ void menu_mmu2_load_filament() {
   START_MENU();
   BACK_ITEM(MSG_MMU2_MENU);
   ACTION_ITEM(MSG_MMU2_ALL, action_mmu2_load_all);
-  ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ _mmu2_load_filament(0); });
-  ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ _mmu2_load_filament(1); });
-  ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ _mmu2_load_filament(2); });
-  ACTION_ITEM_N(4, MSG_MMU2_FILAMENT_N, []{ _mmu2_load_filament(3); });
-  ACTION_ITEM_N(5, MSG_MMU2_FILAMENT_N, []{ _mmu2_load_filament(4); });
+  ACTION_ITEM_N(0, MSG_MMU2_FILAMENT_N, []{ _mmu2_load_filament(0); });
+  ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ _mmu2_load_filament(1); });
+  ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ _mmu2_load_filament(2); });
+  ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ _mmu2_load_filament(3); });
+  ACTION_ITEM_N(4, MSG_MMU2_FILAMENT_N, []{ _mmu2_load_filament(4); });
   END_MENU();
 }
 
 void menu_mmu2_load_to_nozzle() {
   START_MENU();
   BACK_ITEM(MSG_MMU2_MENU);
-  ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ action_mmu2_load_filament_to_nozzle(0); });
-  ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ action_mmu2_load_filament_to_nozzle(1); });
-  ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ action_mmu2_load_filament_to_nozzle(2); });
-  ACTION_ITEM_N(4, MSG_MMU2_FILAMENT_N, []{ action_mmu2_load_filament_to_nozzle(3); });
-  ACTION_ITEM_N(5, MSG_MMU2_FILAMENT_N, []{ action_mmu2_load_filament_to_nozzle(4); });
+  ACTION_ITEM_N(0, MSG_MMU2_FILAMENT_N, []{ action_mmu2_load_filament_to_nozzle(0); });
+  ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ action_mmu2_load_filament_to_nozzle(1); });
+  ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ action_mmu2_load_filament_to_nozzle(2); });
+  ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ action_mmu2_load_filament_to_nozzle(3); });
+  ACTION_ITEM_N(4, MSG_MMU2_FILAMENT_N, []{ action_mmu2_load_filament_to_nozzle(4); });
   END_MENU();
 }
 
@@ -104,11 +104,11 @@ void action_mmu2_unload_filament() {
 void menu_mmu2_eject_filament() {
   START_MENU();
   BACK_ITEM(MSG_MMU2_MENU);
-  ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(0); });
-  ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(1); });
-  ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(2); });
-  ACTION_ITEM_N(4, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(3); });
-  ACTION_ITEM_N(5, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(4); });
+  ACTION_ITEM_N(0, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(0); });
+  ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(1); });
+  ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(2); });
+  ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(3); });
+  ACTION_ITEM_N(4, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(4); });
   END_MENU();
 }
 
@@ -146,11 +146,11 @@ void menu_mmu2_choose_filament() {
   #if LCD_HEIGHT > 2
     STATIC_ITEM(MSG_MMU2_CHOOSE_FILAMENT_HEADER, SS_CENTER|SS_INVERT);
   #endif
-  ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ action_mmu2_choose(0); });
-  ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ action_mmu2_choose(1); });
-  ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ action_mmu2_choose(2); });
-  ACTION_ITEM_N(4, MSG_MMU2_FILAMENT_N, []{ action_mmu2_choose(3); });
-  ACTION_ITEM_N(5, MSG_MMU2_FILAMENT_N, []{ action_mmu2_choose(4); });
+  ACTION_ITEM_N(0, MSG_MMU2_FILAMENT_N, []{ action_mmu2_choose(0); });
+  ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ action_mmu2_choose(1); });
+  ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ action_mmu2_choose(2); });
+  ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ action_mmu2_choose(3); });
+  ACTION_ITEM_N(4, MSG_MMU2_FILAMENT_N, []{ action_mmu2_choose(4); });
   END_MENU();
 }
 

--- a/Marlin/src/lcd/menu/menu_mmu2.cpp
+++ b/Marlin/src/lcd/menu/menu_mmu2.cpp
@@ -63,22 +63,22 @@ void menu_mmu2_load_filament() {
   START_MENU();
   BACK_ITEM(MSG_MMU2_MENU);
   ACTION_ITEM(MSG_MMU2_ALL, action_mmu2_load_all);
-  ACTION_ITEM(MSG_MMU2_FILAMENT0, []{ _mmu2_load_filament(0); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT1, []{ _mmu2_load_filament(1); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT2, []{ _mmu2_load_filament(2); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT3, []{ _mmu2_load_filament(3); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT4, []{ _mmu2_load_filament(4); });
+  ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ _mmu2_load_filament(0); });
+  ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ _mmu2_load_filament(1); });
+  ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ _mmu2_load_filament(2); });
+  ACTION_ITEM_N(4, MSG_MMU2_FILAMENT_N, []{ _mmu2_load_filament(3); });
+  ACTION_ITEM_N(5, MSG_MMU2_FILAMENT_N, []{ _mmu2_load_filament(4); });
   END_MENU();
 }
 
 void menu_mmu2_load_to_nozzle() {
   START_MENU();
   BACK_ITEM(MSG_MMU2_MENU);
-  ACTION_ITEM(MSG_MMU2_FILAMENT0, []{ action_mmu2_load_filament_to_nozzle(0); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT1, []{ action_mmu2_load_filament_to_nozzle(1); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT2, []{ action_mmu2_load_filament_to_nozzle(2); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT3, []{ action_mmu2_load_filament_to_nozzle(3); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT4, []{ action_mmu2_load_filament_to_nozzle(4); });
+  ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ action_mmu2_load_filament_to_nozzle(0); });
+  ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ action_mmu2_load_filament_to_nozzle(1); });
+  ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ action_mmu2_load_filament_to_nozzle(2); });
+  ACTION_ITEM_N(4, MSG_MMU2_FILAMENT_N, []{ action_mmu2_load_filament_to_nozzle(3); });
+  ACTION_ITEM_N(5, MSG_MMU2_FILAMENT_N, []{ action_mmu2_load_filament_to_nozzle(4); });
   END_MENU();
 }
 
@@ -104,11 +104,11 @@ void action_mmu2_unload_filament() {
 void menu_mmu2_eject_filament() {
   START_MENU();
   BACK_ITEM(MSG_MMU2_MENU);
-  ACTION_ITEM(MSG_MMU2_FILAMENT0, []{ _mmu2_eject_filament(0); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT1, []{ _mmu2_eject_filament(1); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT2, []{ _mmu2_eject_filament(2); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT3, []{ _mmu2_eject_filament(3); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT4, []{ _mmu2_eject_filament(4); });
+  ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(0); });
+  ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(1); });
+  ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(2); });
+  ACTION_ITEM_N(4, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(3); });
+  ACTION_ITEM_N(5, MSG_MMU2_FILAMENT_N, []{ _mmu2_eject_filament(4); });
   END_MENU();
 }
 
@@ -146,11 +146,11 @@ void menu_mmu2_choose_filament() {
   #if LCD_HEIGHT > 2
     STATIC_ITEM(MSG_MMU2_CHOOSE_FILAMENT_HEADER, SS_CENTER|SS_INVERT);
   #endif
-  ACTION_ITEM(MSG_MMU2_FILAMENT0, []{ action_mmu2_choose(0); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT1, []{ action_mmu2_choose(1); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT2, []{ action_mmu2_choose(2); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT3, []{ action_mmu2_choose(3); });
-  ACTION_ITEM(MSG_MMU2_FILAMENT4, []{ action_mmu2_choose(4); });
+  ACTION_ITEM_N(1, MSG_MMU2_FILAMENT_N, []{ action_mmu2_choose(0); });
+  ACTION_ITEM_N(2, MSG_MMU2_FILAMENT_N, []{ action_mmu2_choose(1); });
+  ACTION_ITEM_N(3, MSG_MMU2_FILAMENT_N, []{ action_mmu2_choose(2); });
+  ACTION_ITEM_N(4, MSG_MMU2_FILAMENT_N, []{ action_mmu2_choose(3); });
+  ACTION_ITEM_N(5, MSG_MMU2_FILAMENT_N, []{ action_mmu2_choose(4); });
   END_MENU();
 }
 

--- a/Marlin/src/lcd/menu/menu_mmu2.cpp
+++ b/Marlin/src/lcd/menu/menu_mmu2.cpp
@@ -39,7 +39,7 @@ void _mmu2_load_filamentToNozzle(uint8_t index) {
   ui.reset_status();
   ui.return_to_status();
   ui.status_printf_P(0,  GET_TEXT(MSG_MMU2_LOADING_FILAMENT), int(index + 1));
-  if (mmu2.load_filament_toùozzle(index)) ui.reset_status();
+  if (mmu2.load_filament_to_nozzle(index)) ui.reset_status();
 }
 
 inline void action_mmu2_load_filament_to_nozzle(const uint8_t tool) {

